lib/ledger.js

```diff
@@ -112,7 +113,7 @@ class LedgerBTCApp {
    * @throws {LedgerError}
    */
 
-  async hashTransactionStart(tx, tis = {}, isNew, hasWitness) {
+  async hashTransactionStart(tx, tis = new Map(), isNew, hasWitness) {
```

Comment:
this function is only called with one argument sometimes? maybe follow the way its done in:

`hashTransactionStartNullify`

```
if (!tis)
  tis = new Map();
```

It feels strange to me having a default arg when its not the last argument/s but maybe thats just a personal preference. Python `kwargs` solve that problem well imo